medmodels/statistic_evaluations/statistical_analysis/inferential_statistics.pyi

```diff
@@ -0,0 +1,24 @@
+from typing import List, Tuple
+
+from numpy.typing import ArrayLike
+
+from medmodels.statistic_evaluations.comparer.data_comparer import TestSummary
+
+def test_for_normal_distribution(sample: ArrayLike) -> bool: ...
+def decide_hypothesis_test(samples: List[ArrayLike], alpha: float) -> TestSummary: ...
```

Comment:
Like it automatically chooses the test?

Reply:
Yes basically, it will be necessary for the data comparer where different attribute types and distributions are possible. In this function it will basically be a lot of criteria and then from there the right test will be chosen.

Reply:
Okay, I will approve for now, but maybe I will suggest name change in the future!